cfme/tests/containers/test_containers_overview_data_integrity.py

```diff
@@ -81,15 +75,39 @@ def test_containers_overview_data_integrity():
     navigate_to(ContainersOverview, 'All')
     # We should wait ~2 seconds for the StatusBox population
     # (until we find a better solution)
+    # Since we collect images from Openshift and from the pods,
+    # images are tested separately
     time.sleep(2)
     statusbox_values = {data_set.object: int(StatusBox(data_set.name).value())
                         for data_set in DATA_SETS}
-    api_values = get_api_object_counts(list_providers_by_class(ContainersProvider))
+    api_values = get_api_object_counts(
+        list_providers_by_class(ContainersProvider))
+
+    list_img_from_registry = provider.mgmt.list_image()
+    list_img_from_registry_splitted = [i[2].split(
+        '@sha256:')[-1] for i in list_img_from_registry]
+
+    list_img_from_openshift = provider.mgmt.list_image_openshift()
+    list_img_from_openshift_splitted = [d['name']
+                                        for d in list_img_from_openshift]
+    list_img_from_openshift_parsed = [i[7:].split(
```

Comment:
Well it seems `re` wouldn't help much, but I would suggest that instead of using `i[2]` to use `i.id`.